debian/snuffleupagus.install

```diff
@@ -0,0 +1,2 @@
+config/default.rules /etc/snuffleupagus/
+scripts /usr/lib/nbs/
```

Comment:
I think that we may want to put it somewhere else if we want to merge this PR :)